src/chi/components/input-number/_input-wrapper.scss

```diff
@@ -168,6 +183,16 @@ $m-inputNumber-width: (
 
     & > button {
       color: $brand-color;
+      display: block;
+      height: map-get(map-get($m-inputNumber, md), button-height);
+
+      & > div.a-btn__content {
+        display: block;
```

Comment:
@dani-cl-madrid please remove this property as it is not longer necessary with the height and the display block in the button(l.186)